openBCIBoard.js

```diff
@@ -66,6 +67,12 @@ function OpenBCIFactory () {
   *              `v1` - Firmware Version 1 (Default)
   *              `v2` - Firmware Version 2
   *
+  *     - `simulatorFragmentation` {String} - Specifies how to break packets to simulate fragmentation.
+  *          3 Possible Options:
+  *              `Random` - output randomly-lengthed chunks of data (Default)
```

Comment:
hmm I picked uppercase to match the convention used in line noise
should line noise be lowercase?  notably if this used `none` it would be weird with linenoise using `None`